DecipherNow/gm-fabric-dashboard
# 1064 browserslist

Resolves #1064 

## Proposed Changes

  - Pull `babel-preset-react-app` logic into a local file and modify to contain our plugins and presets
  - Add browserslist logic with out browser support matrix, and configure babel env and PostCSS autoprefixer to use the file. Unfontunately, babel env cannot use the browserslist value in `package.json` until babel 7.x is released, so I've had to implement the list separately as a node module.
  
## How to Test

  - Run the app locally and test in our supported browsers via BrowserStack
  - Run Storybook
  - Run `npm test`
  - Run `npm run build` and confirm that the build process completes.
